extensions/getsound/README.md

```diff
@@ -0,0 +1,48 @@
+
+![IMAGE 2023-07-01 15:02:59](https://getsound.ai/getsound/images/RayExtention_Icon_512.png)
+
+
+# GetSound - Raycast Extension
+
+## Overview
+
+[GetSound.ai](http://getsound.ai/) Raycast Extension is a slimmed-down version of our main GetSound.ai app, crafted especially for Raycast Community. It provides developers and users with a convenient and straightforward way to take a break and refocus during their work. Simply type "Focus" and you can generate short, 5-10 minute, calming ambient music tracks to help you get back on track.
+
+**What makes the full version of GetSound different?**
+
+If you want to experience the full GetSound.ai immersive audio user experience, we suggest downloading the app from our [Website](https://getsound.ai). 
+Unlike the slimmed-down Raycast version, the full app features GetSound's unique RTS (Real-Time Soundscape) algorithm, which creates ambient music based on your environment. Factors such as weather, location, and time of day all contribute to creating a unique soundscape in that moment. So, if you want to meditate, relax, or stay inside on a rainy day, GetSound will generate the perfect soundtrack and keep playing new music for as long as you like. Plus, you'll enjoy these key features:
+* Enjoy continuous, uninterrupted and non-repeated soundscape music 24/7.
+* Hi-resolution audio quality.
+* Add and control extra layers of environmental effects, such as rivers, oceans, lakes, waterfalls, coffee shops, and more.
+* Change your location to generate more unique soundscapes.
+* Refresh funtion to regenerate new real-time soundscapes.
+* Weather Sync - This feature will be available only when certain weather conditions, like rain, storms, and winds, occur at your location. You can easily adjust the volume to your liking.
+* Timer Session - Do you have limited time to meditate or relax? With this feature, you can set the music to fade out after a set amount of time. Perfect for those with 15 minutes or less to spare.
+
+## Installation
+
+Install the [GetSound.ai](http://getsound.ai/) Raycast Extension from the Raycast Store.
+After installation, you can access the extension by typing "Focus" in the Raycast command bar.
+
+## Commands
+
+* **Quick Access**: Just type "Focus" to start your 5-10 minutes relaxing ambient music track.<br>
```

Comment:
Im confused about "5-10" minutes. Is this configurable?

Reply:
> Im confused about "5-10" minutes. Is this configurable?

It is generative. Track length is about 5-10 minutes. Its not configurable yet, but we are already thinking about it

Reply:
ah ok makes sense, thanks!